ols/constants.py

```diff
@@ -83,40 +90,42 @@
 Response:
 """
 
-QUESTION_VALIDATOR_PROMPT_TEMPLATE = """
+# we are escaping the curly braces there so that we can use the format
+# function to fill query later
```

Comment:
I don't understand this comment.....aren't the now-bracketed values just immediately getting substituted by the constant values?  (which is fine, it's a good change to use the constants, but i don't understand the part about using the format function later)

Reply:
@onmete ^^^